xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1030 @@
+     </step>
+    </procedure>
+   </sect3>
+   <sect3>
+    <title>Client Configuration</title>
+    <para>
+     Add the webserver as a repo to <command>helm</command> on the admin node.
+     </para>
+<screen>&prompt.user;<command>helm repo add SUSE-MIRROR <replaceable>&lt;internal-mirror.hostname&gt;</replaceable></command>
+   </screen>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.update">
+    <title>Updating Registry Mirror And Helm Charts</title>
+    <note>
+     <title>Live Update Of Registry</title>
+     <para>
+      There is no need to stop the Docker registry services while doing the
+      backup and restore procedures. All changed images will be re-indexed
+      automatically.
+     </para>
+    </note>
+    <para>
+     Helm charts and container images must be refreshed in the same procedure,
+     otherwise charts might refer to image versions that are not mirrored or you
+     are mirroring outdated image versions that cause the chart deployment to
+     fail.
+    </para>
+
+    <procedure>
+     <title>Pull Data From Upstream Sources</title>
+     <step>
+      <para>
+       SSH into the mirror server on the external network.
+      </para>
+     </step>
+     <step>
+      <para>
+      Download all charts from the repository to the filesystem
+      (e.g. <filename>/tmp/charts</filename>).
+      </para>
+<screen>&prompt.user;<command>helm-mirror https://kubernetes-charts.suse.com <replaceable>/tmp/charts</replaceable></command>
+     </screen>
+    </step>
+    <step>
+     <para>
+      Translate the chart information into the <literal>skopeo</literal> format.
+     </para>
+<screen>&prompt.user;<command>helm-mirror inspect-images <replaceable>/tmp/charts</replaceable> -o skopeo --file-name <replaceable>sync.yaml</replaceable></command></screen>
+    </step>
+    <step>
+     <para>
+      Download all the referenced images using <command>skopeo</command>
+     </para>
+<screen>&prompt.user;<command>skopeo sync --source-yaml <replaceable>sync.yaml</replaceable> dir:<replaceable>/tmp/skopeodata</replaceable></command>
+     </screen>
+     <para>
+      <command>skopeo</command> will automatically create a directory named
+      after the hostname of the registry from which you are downloading the
+      images. The final path will be something like <filename>/tmp/skopeodata/registry.suse.com/</filename>.
+     </para>
+    </step>
+    <step>
+     <para>
+      Populate the local registry with the downloaded data.
+     </para>
+     <para>
+      For <literal>--dest-creds</literal> you must use the credentials you
+      created during <xref linkend="sec.deploy.scenarios.airgap.container-registry.mirror" />.
+     </para>
+<screen>&prompt.user;<command>skopeo sync --dest-creds <replaceable>USERNAME:PASSWORD</replaceable> \
+dir:<replaceable>/tmp/skopeodata/registry.suse.com/</replaceable> <replaceable>docker://myregistry.local</replaceable></command>
+      </screen>
+    </step>
+    <step>
```

Comment:
I would add another step to remove the `/tmp/skopeodata` directory. This is no longer needed and it's wasting quite some disk space.

Reply:
done